### ISSUE TITLE
Highlight coordinate scaling in documentation

### ISSUE THREAD

**Issue by b-steve** [nimble-dev/nimbleSCR#1]
Thanks for the wonderful SCR functions!

Just a suggestion: the requirement to scale locations could be made a little more explicit in the documentation for functions like `dpoisLocal_normal`? Perhaps wording like this in the argument description: 

"Matrix of **scaled** x- and y-coordinates of all traps **(see Details below).**"

At the moment the argument definitions themselves don't mention the scaling; you need to look down in the Details section to spot it. Also, in the Details section, it's very explicit that the trap locations need to be scaled, but there's no mention of scaling activity center locations. I think it's also worth highlighting that the activity center locations need to be provided in the scaled coordinate system too. For example, something like this:

"The `dpoisLocal_normal` distribution requires x- and y- detector coordinates (`trapCoords`) **and activity center corrdinates (`s`)** to be scaled to the habitat grid (`habitatGrid`) using the (`scaleCoordsToHabitatGrid` function.)"

I know it's sensible enough to expect the user to provide both trap and activity centers in the same coordinate system, but we recently made the error of accidentally scaling only the trap locations, and it took us quite a while to track down the bug.

**Comment by weizhangstats** [comment]
Hi Ben @b-steve, glad that you are interested in the package. I just jump in to say hi. :)

**Comment by soumenstat89** [comment]
Hi Ben @b-steve ,
Sorry for the lack of clear documentation. We agree that we should have been clearer. We will now adjust the documentation to mention that both traps and activity centers should be scaled to the habitat grid as you suggested. Note that we are also planning to make a function to create initial values (scaled) for the activity center locations based on the pattern of individual detection. That should also help users fitting SCR models. Thanks for using nimbleSCR and please keep letting us know how we can improve nimbleSCR.

**Comment by b-steve** [comment]
@soumenstat89, thanks for the reply. That sounds great! No need to apologise. The model we're fitting isn't terribly new or exciting; it's just using an inhomogeneous Poisson process rather than the standard homogeneous one. This is easy enough to fit with maximum likelihood but turned out to be much tricker than we expected using MCMC. We spent ages trying to get JAGS working, but the mixing was so atrocious that we gave up. Without these new NIMBLE functions we probably wouldn't have managed it, so thanks!

@wzhang721, hello, and good to hear from you!

**Comment by Cyril-Milleret** [comment]
Hi @b-steve, I have added your suggestions to the description of the function. 
In the next version of nimbleSCR (0.1.3; should be available on CRAN in a couple of days), we have also added functions to fit SCR models with an inhomogeneous Poisson process. You would need to use dbernppAC() for the placement of individual activity centers. if you are interested, we have made a vignette [here](https://github.com/nimble-dev/nimbleSCR/blob/master/nimbleSCR/vignettes/Point_Process.Rmd) with an example on how to fit such models.
